plugins/rbac-backend/src/service/permission-policy.ts

```diff
@@ -343,4 +292,23 @@ export class RBACPermissionPolicy implements PermissionPolicy {
     }
     return false;
   }
+
+  private isAuthorized = async (
+    userIdentity: string,
+    permission: string,
+    action: string,
+  ): Promise<boolean> => {
+    const filter: string[] = [userIdentity, permission, action];
+    if (this.policiesFile) {
+      await loadFilteredCSV(
```

Comment:
Could you apply try catch with log error, just in case unexpected error?

Reply:
Question, wouldn't the above [try catch](https://github.com/janus-idp/backstage-plugins/blob/eb6042c38c9993e0ff43086e81c039aa91d75a71/plugins/rbac-backend/src/service/permission-policy.ts#L219:L277) handle any potential errors that might be thrown from `loadFilteredCSV` since `authorize` is called within that try catch?